lib/confirmation.rb

```diff
@@ -0,0 +1,18 @@
+require_relative "takeaway.rb"
+
+class Confirmation 
+  #Confirmation --> dishes_selected --> selected_dish_total --> confirmed 
+  #Confirmation --> confirmed --> text_sent 
+  #--> "Thank you! Your order was placed and will be delivered before 18:52"
+  #Method without Twilio API 
+  #Dummy code unfinished 
+  
```

Comment:
Will you be using the Takeaway class as  a dependency here? i.e. initialising `Confirmation` by passing (for example) `takeaway = Takeaway.new` into `initialize` for this class?